src/widgets/vitals/vitals-header/vital-header-details.component.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import styles from "./vital-header-details.component.scss";
+
+interface VitalHeaderStateDetailsProps {
+  unitName: string;
+  value: string | number;
+  unitSymbol: string | React.ReactNode;
```

Comment:
I think React.ReactChild is essentially `string | ReactNode` (it also contains boolean and number, but that should  not matter much).